geolocator_platform_interface/lib/src/enums/location_accuracy_status.dart

```diff
@@ -0,0 +1,8 @@
+/// Represent the current Location Accuracy Status on iOS devices
+enum LocationAccuracyStatus {
+  /// A approximate location will be returned (Approximate location)
+  reduced,
+
+  /// The precise location of the device will be returned
```

Comment:
nit: missing period:

```suggestion
  /// The precise location of the device will be returned.
```